common/src/ethereum.rs

```diff
@@ -236,7 +262,7 @@ where
     M: Middleware + 'static,
 {
     let receipt = loop {
-        let spinner = term::spinner("Waiting for transaction to be signed...");
+        let spinner = term::spinner("Waiting for transaction to be signed & sent...");
```

Comment:
I would revert this, we're not waiting for it to be sent, since the sending is instant once it is signed, and once it's signed we log that it's been sent.

Reply:
The reason I changed this is because when using `WalletConnect` there's a delay between when you sign tx in walletconnect and when it's sent back to app to be sent to blockchain. When there was an error with tx info, e.g. gas being too low, it'd still say "Waiting for transaction to be signed..." failed which is not very helpful, that's why I added "... signed & sent". Makes sense?

Reply:
Ah I see, that's interesting, if there's no way to wait on the intermediate step, then I'd just say `Waiting for transaction to be submitted`

Reply:
Signing and sending both happens inside `ethers` which we can't intercept ([code](https://github.com/gakonst/ethers-rs/blob/master/ethers-middleware/src/signer.rs#L256-L264)).